src/Spatial/Euclidean/Point3D.cs

```diff
@@ -226,6 +226,16 @@ public static Point3D IntersectionOf(Plane plane, Ray3D ray)
             return plane.IntersectionWith(ray);
         }
 
+        public static Point3D operator +(Vector3D v, Point3D p)
```

Comment:
For some reason I did not add these, point + vector felt natural, vector + point did not (for me).
Maybe this can be a separate PR to give others the opportunity to have a look?